public/index.html

```diff
@@ -1,13 +1,14 @@
 <!DOCTYPE html>
   <head>
+      <link href="https://fonts.googleapis.com/css?family=Nunito+Sans&display=swap" rel="stylesheet">
     <meta charset="utf-8" />
     <title>Reviews</title>
   </head>
   <body>
-    <h1>Reviews</h1>
 
     <div id="app">React Aint Workin</div>
 
-    <script src="bundle.js"></script>
+    <script src="main.js"></script>
+    <link rel="stylesheet" type="text/css" href="style.css">
```

Comment:
I believe since you are using CSS module, you can remove this line.

Reply:
I removed it, good catch!